runtime/ts/plan/plan-producer.ts

```diff
@@ -36,8 +36,9 @@ export class PlanProducer {
   search: string;
   searchStore: StorageProviderBase;
   searchStoreCallback: ({}) => void;
+  devTools = false;
 
-  constructor(arc: Arc, store: StorageProviderBase, searchStore: StorageProviderBase) {
+  constructor(arc: Arc, store: StorageProviderBase, searchStore: StorageProviderBase, {devTools = false} = {}) {
```

Comment:
I added the additional parameter because when I discussed it with @shans he recommended we didn't store the generations by default, because of how significantly larger they are (than everything else we store), so we need an additional means to control that, other than whether devtools are connected.
I realize now the parameter name should be more specific then, and also I still need to add a url param to the shell and flag to remote planner to really turn it on/off.

Reply:
Ok, it all makes sense.

To be figured out exactly how to control turning on/off the storage of generations. Thanks!